README.md

```diff
@@ -611,3 +495,75 @@ Related open source projects are:
 - SEGGER and J-LINK are registered trademarks of SEGGER Microcontroller GmbH.  
 - Node.js is a registered trademark of the OpenJS Foundation.  
 - GDB and GCC are part of the GNU Project and are maintained by the Free Software Foundation.  
+
+-----
```

Comment:
If the following isn't a backup of originally used text, then you may want to move it before the Trademarks section.
Note that the above "Pseudo Debugger Types" section will go away with the default launch configs with #318